src/dawn/engine_dawn.h

```diff
@@ -36,7 +37,11 @@ class EngineDawn : public Engine {
 
   // Engine
   // Initialize with a default device.
-  Result Initialize() override;
+  Result Initialize(const Format* color_frame_format,
```

Comment:
This is going to have to change for amberscript as the format will be provided as colour attachments later, I'd prefer if we moved in that direction and don't treat framebuffer info as a 'requirement'